config/src/main/java/com/alibaba/fescar/config/ZookeeperConfiguration.java

```diff
@@ -61,9 +61,13 @@
 
     public ZookeeperConfiguration() {
         if (zkClient == null) {
-            zkClient = new ZkClient(FILE_CONFIG.getConfig(FILE_CONFIG_KEY_PREFIX + SERVER_ADDR_KEY),
-                FILE_CONFIG.getInt(FILE_CONFIG_KEY_PREFIX + SESSION_TIME_OUT_KEY),
-                FILE_CONFIG.getInt(FILE_CONFIG_KEY_PREFIX + CONNECT_TIME_OUT_KEY));
+            synchronized (ZookeeperConfiguration.class) {
+                if (null == zkClient) {
+                    zkClient = new ZkClient(FILE_CONFIG.getConfig(FILE_CONFIG_KEY_PREFIX + SERVER_ADDR_KEY),
+                        FILE_CONFIG.getInt(FILE_CONFIG_KEY_PREFIX + SESSION_TIME_OUT_KEY),
```

Comment:
SESSION_TIME_OUT and CONNECT_TIME_OUT_KEY add config default value

Reply:
ok.

Reply:
done